extension/lib/doc.js

```diff
@@ -172,8 +173,8 @@ class Doc {
             let wordRegex = /\b\w+\b/g;
             let wordList = text.match(wordRegex);
             for (var i in wordList) {
-                let word = wordList[i];
-                if (stop_words.has(wordList[i].toLowerCase())) { // TODO: REFACTOR AS SET; same for other instances like this in code
+                let word = wordList[i].toLowerCase();
+                if (stop_words.has(word)) { // TODO: REFACTOR AS SET; same for other instances like this in code
                     // Don't include stop words
                 }
                 else if (termFreq[wordList[i]]) { // Word is in termFrequency dict
```

Comment:
hm do u not want to use the lower case word for ur stats as well?

Reply:
This should just handle everything lower case, if I am correct?

Reply:
This line: (termFreq[wordList[i]]) {
Probably is termFreq[word] ?

And make sure to check for similar bugs elsewhere too

Reply:
termFreq[wordList[i]]) -> termFreq[word]?

And if the above is correct, pls check other places for similar bugs

Reply:
Oh I see; misinterpreted line of comment, yes you're right